packages/formatters/src/github-actions.ts

```diff
@@ -0,0 +1,41 @@
+import { DiagnosticSeverity, Dictionary } from '@stoplight/types';
+import { Formatter } from './types';
+
+const OUTPUT_TYPES: Dictionary<string, DiagnosticSeverity> = {
+  [DiagnosticSeverity.Error]: 'error',
+  [DiagnosticSeverity.Warning]: 'warning',
+  [DiagnosticSeverity.Information]: 'notice',
+  [DiagnosticSeverity.Hint]: 'notice',
+};
+
+type OutputParams = {
+  title?: string;
+  file: string;
+  col?: number;
+  endColumn?: number;
+  line?: number;
+  endLine?: number;
+};
+
+export const githubActions: Formatter = results => {
+  const path = require('path') as { relative: (from: string, to: string) => string };
```

Comment:
```ts
import * as path form '@stoplight/path
```
could we use `@stoplight/path`?

Reply:
Fixed in [a18d591](https://github.com/stoplightio/spectral/pull/2508/commits/a18d5917afa10aa5f95110384cd6b413d8168c93).